sensor_array/analysis/brute_force_analysis.py

```diff
@@ -641,6 +643,150 @@ def save_unbinned_array_pmf_data(gases, list_of_arrays, list_of_array_ids, all_a
             for line in pmf_data_alt:
                 writer.writerow(line)
 
+def prepare_ternary_dict_rgba(a,b,c,z,vmin,vmax,cmap):
+    """
+    Prepare data for use with 'ternary'
+    """
+
+    color_norm = matplotlib.colors.Normalize(vmin=vmin, vmax=vmax)
+    alpha_norm = matplotlib.colors.Normalize(vmin=vmin-(vmax-vmin), vmax=vmax)
+    color_map = matplotlib.cm.get_cmap(cmap)
+
+    data_dict_rgba = dict()
+    data_dict_zval = dict()
+    for i in range(len(a)):
+        A_out = np.round(a[i]*100,3)
+        B_out = np.round(b[i]*100,3)
+        C_out = np.round(c[i]*100,3)
```

Comment:
variables are not normally capitalized: https://www.python.org/dev/peps/pep-0008/#id45